openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -0,0 +1,30 @@
+{% spaceless %}
+{% load i18n %}
+
+{% if file_upload_type %}
+    {% if header %}
+        <header class="{{ class_prefix }}__display__header">
+            <h3 class="{{ class_prefix }}__display__title">
+                {% trans header %}
+            </h3>
+        </header>
+    {% endif %}
+
+    <div class="{{ class_prefix }}__display__file {% if not file_url %}is--hidden{% endif %}" id="submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
+        {% if file_upload_type == "image" %}
+            <img id="submission__answer__file"
+                 class="submission--image"
+                 alt="{% trans "The image associated with this submission." %}"
+                 src="{{ file_url }}" />
+        {% elif file_upload_type == "pdf-and-image" or file_upload_type == "custom" %}
+            <a href="{{ file_url }}" id="submission__answer__file" class="submission--file" target="_blank">
+                {% trans "View the file associated with this submission." %}
+            </a>
+            {% if show_warning %}
+                <p class="submission_file_warning">{% trans "(Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed or endorsed by edX. If you decide to access it, you do so at your own risk.)" %}</p>
```

Comment:
@stvstnfrd sure, no problem. This issue was raised in August 2015, and addressed at that time by the members of the edX legal team as mentioned above. Two members of the edX open source team were copied on the entire correspondence, @sarina and @nedbat. The wording suggested at that time was, in fact:
 "Caution:  This file was uploaded by another course learner and has not been verified, screened, approved, reviewed or endorsed by edX [or the institution offering the course?].  If you decide to access it, you do so at your own risk. " 
When this issue came up again yesterday, I commented above before looking up the complete email thread. As soon as possible afterward, I did do the research, realized my error, and IM'd @andy-armstrong to make sure that he got the correct info as soon as possible and could act accordingly. 
While I cannot comment on how long a patch might take to put in place, I hope that we can agree that all will, in fact, be well.